README.md

```diff
@@ -37,7 +37,7 @@
 | [Satori-SDK-Go](https://github.com/dezhishen/satori-sdk-go)    | [Dezhishen](https://github.com/Dezhishen)  | Golang |
 | [Satori.NET](https://github.com/bsdayo/Satori.NET)             | [bsdayo](https://github.com/bsdayo) | .NET / C# |
 | [SPPS](https://github.com/im-patory/spps) w/ [Patory](https://github.com/im-patory/patory) | [Lipraty](https://github.com/Lipraty) | PHP |
-| [YurnSatoriSdk](https://github.com/Nyayurn/YurnSatoriSdk)      | [Nyayurn](https://github.com/Nyayurn) | Java |
+| [YurnSatoriSdk](https://github.com/Nyayurn/YurnSatoriSdk)      | [Nyayurn](https://github.com/Nyayurn) | Jvm / Java |
```

Comment:
I think Java refers the JVM obviously, so you don't need to mention it again.
Besides, it should be `JVM` all uppercased rather than capitalised.

我想爪哇很明显指向JVM，所以你不需要提到它再次。
另外，它应该是`JVM`全大写的而非首字母大写。

Reply:
类似上面 .NET / C# 一样, 带上 JVM 表明这个框架支持整个 JVM 生态(如 Kotlin)而非仅支持 Java

Reply:
The ‘Language’ here should only indicate the language that the SDK is written in, rather than the language outside to invoke the SDK.

Since kotliner always knows that he wants a Java or Kotlin or even Clojure SDK.